### ISSUE TITLE
Navigation doesn’t have background on mobile

### ISSUE THREAD

**Issue by jancborchardt** [nextcloud/nextcloud.com#343]
On mobile, opening the navigation results in this:
![capture du 2017-01-17 13-30-18](https://cloud.githubusercontent.com/assets/925062/22020582/557e328a-dcb9-11e6-91e5-3c51cc59fe77.png)

**Comment by Espina2** [comment]
I don't see this error happening. Can you do a hard refresh and try again?

**Comment by jancborchardt** [comment]
Yeah, did that already with both Firefox and Chrome, incognito window, hard cache refresh and all. @nextcloud/website can anyone of you reproduce too?

**Comment by MariusBluem** [comment]
Chrome working for me 😱

**Comment by Espina2** [comment]
It works here, even in edge mobile. :p

**Comment by jancborchardt** [comment]
Just reproduced today independently on an iPhone with Safari, as well as on Ubuntu with Firefox and Chrome.

**Comment by Espina2** [comment]
This are fixed here: https://github.com/nextcloud/nextcloud.com/tree/menu-mobile
Just not merged yet

**Comment by jancborchardt** [comment]
@Espina2 @jospoortvliet can we merge that then? The site is broken on mobile which is not acceptable.

**Comment by Espina2** [comment]
this is already merged.

**Comment by Espina2** [comment]
@jancborchardt What is broken in mobile? I didn't see anything.
